pkgs/applications/networking/browsers/vimb/default.nix

```diff
@@ -1,18 +1,20 @@
-{ stdenv, fetchurl, pkgconfig, libsoup, webkit, gtk2, glib-networking
+{ stdenv, fetchFromGitHub, pkgconfig, libsoup, webkit, gtk3, glib-networking
 , gsettings-desktop-schemas, makeWrapper
 }:
 
 stdenv.mkDerivation rec {
   name = "vimb-${version}";
-  version = "3.1.0";
+  version = "3.3.0";
 
-  src = fetchurl {
-    url = "https://github.com/fanglingsu/vimb/archive/${version}.tar.gz";
-    sha256 = "1gws028c2v1zh6r142hmjvi2m447lwqqh65m6z3dzcar2yw35z3f";
+  src = fetchFromGitHub {
+    owner = "fanglingsu";
+    repo = "vimb";
+    rev = version;
+    sha256 = "1qg18z2gnsli9qgrqfhqfrsi6g9mcgr90w8yab28nxrq4aha6brf";
   };
 
-  nativeBuildInputs = [ pkgconfig ];
-  buildInputs = [ makeWrapper gtk2 libsoup webkit gsettings-desktop-schemas ];
+  nativeBuildInputs = [ makeWrapper pkgconfig ];
+  buildInputs = [ gtk3 libsoup webkit gsettings-desktop-schemas ];
```

Comment:
I just noticed the `makeWrapper`:
If you add `nativeBuildInputs = [ wrapGAppsHook ];` and `buildInputs = [ glib gnome3.gsettings_desktop_schemas ];` then you don't `makeWrapper` and the result 
is less fragile.
https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/networking/instant-messengers/rambox/default.nix#L33